File: 1512-number-of-good-pairs/1512-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        ans = 0

        # i,j 차례대로 for문 돌면서,
        for i in range(len(nums)):
            for j in range(len(nums)):
                # 조건에 맞으면 ans++
                if i < j and nums[i] == nums[j]:
                    ans += 1
        
        return ans

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.c
int numIdenticalPairs(int* nums, int numsSize) {
    int ans = 0;

    // i,j 차례대로 for문 돌면서,
    for (int i=0; i<numsSize; i++) {
        for (int j=0; j<numsSize; j++) {
            // 조건에 맞으면 ans++
            if ((i < j) && (nums[i] == nums[j])) {
                ans++;
            }
        }
    }

    return ans;
}